lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -81,6 +82,12 @@ protected function configure() : void
                 InputOption::VALUE_OPTIONAL,
                 'Check Database Platform to the generated code.',
                 true
+            )
+            ->addOption(
+                'allow-no-diff',
```

Comment:
How about `allow-empty-diff` instead? `allow-no-diff` can be interpreted as "Not a single diff will be allowed".

Reply:
Hi, thanks for your comments. 
I understand what you saying, but I would prefer to use `allow-no-changes`, what do you think? the rationale is that `allow-empty-diff` may be interpreted as "An empty diff file will be created" and - correct me if I'm wrong - that's what `GenerateCommand` does. Instead this command only displays a message instead of an Exception.

Reply:
I think `allow-no-changes` presents the same issue as `allow-no-diff`, and that `allow-empty-diff` will hardly be interpreted as "An empty diff file will be created"  when the context is the diff command.

Reply:
Ok :)